### ISSUE TITLE
ClassifyThreshold only returns one value

### ISSUE THREAD

**Issue by scaldarola** [Kagami/go-face#46]
Let's say I have an array of faces, and a new one to compare against them, as of today, the function ClassifyThreshold only returns the first possible value matched in the labels that the recognizer has. What if, inside that array of faces there was more than one match to the new face I am trying to classify? That function could be refactored in order to return an array of indexes in the labels array. 

How possible do you think this is?

**Comment by Kagami** [comment]
Yeah, good idea, we may want to return few most probable matches.

**Comment by scaldarola** [comment]
@Kagami I am trying this lib on a pet project, and found a way to reproduce the behaviour described in this issue, but in go code. I think the best solution would be that the c++ handles it. Do you see it as a long shot?
